src/badfish/main.py

```diff
@@ -2052,7 +2217,11 @@ async def execute_badfish(_host, _args, logger, format_handler=None):
     screenshot = _args["screenshot"]
     retries = int(_args["retries"])
     output = _args["output"]
-
+    get_scp_targets = _args["get_scp_targets"]
+    scp_targets = _args["scp_targets"]
```

Comment:
scp_targets = _args.get("scp_targets, "ALL")

Reply:
Sure can change that, although I also see a benefit of a default specified in the function signature.